src/rnn/attention.h

```diff
@@ -11,14 +11,16 @@ Expr attOps(Expr va, Expr context, Expr state);
 
 class GlobalAttention : public CellInput {
 private:
-  Expr Wa_, ba_, Ua_, va_;
+  std::vector<Expr> Was_, bas_, Uas_;
+  std::vector<Expr>vas_;
 
-  Expr gammaContext_;
-  Expr gammaState_;
+  std::vector<Expr> gammaContexts_;
```

Comment:
Please add a comment stating what this vector is indexed with. Can be brief, e.g.
`// [attention head index]`
(or whatever it is).